model_compression_toolkit/ptq/keras/quantization_facade.py

```diff
@@ -134,14 +134,15 @@ def keras_post_training_quantization_experimental(in_model: Model,
 
         fw_impl = KerasImplementation()
 
-        tg, bit_widths_config = core_runner(in_model=in_model,
-                                            representative_data_gen=representative_data_gen,
-                                            core_config=core_config,
-                                            fw_info=fw_info,
-                                            fw_impl=fw_impl,
-                                            tpc=target_platform_capabilities,
-                                            target_kpi=target_kpi,
-                                            tb_w=tb_w)
+        # Ignore returned hessian service as PTQ does not use it
+        tg, bit_widths_config, _ = core_runner(in_model=in_model,
```

Comment:
Why not init a hessian service in this facade? (you do in the pytorch facade, and it might be relevant for parameters selection if I'm not mistaken)